invenio_records_lom/services/schemas/metadata.py

```diff
@@ -25,6 +26,14 @@
 from marshmallow_utils.html import sanitize_unicode
 
 
+def now_isoformat() -> str:
+    """Return current date and time in ISO-8601 format.
+
+    example-return: '2020-06-24T05:34:56.789123+00:00'
```

Comment:
do we need that +00:00? could we use `datetime.now(timezone.utc).replace(tzinfo=None)` instead?

Reply:
and i would prefer to not use any dependency for things that the stdlib could provide

Reply:
agreed

for more context on necessity of `.replace(tzinfo=None)`, see updated PR-message